src/parser/jobs/ast/modules/Synastry.tsx

```diff
@@ -21,24 +24,26 @@ const GCD_ST_HEAL: Array<keyof ActionRoot> = [
 ]
```

Comment:
GH being dumb, we no longer recommend using `Array<keyof ActionRoot>` since it's been replaced by a proper type, `ActionKey` which you can use like `ActionKey[]` for the array.

Reply:
updated